textstat/textstat.py

```diff
@@ -398,6 +397,30 @@ def text_standard(self, text, float_output=None):
                 lower_score, get_grade_suffix(lower_score),
                 upper_score, get_grade_suffix(upper_score)
             )
+    @repoze.lru.lru_cache(maxsize=128)
+    def spache_readability(self, text, float_output=False):
```

Comment:
Most of textstat's functions return a float as default, can we change this to `float_output=True`?

Reply:
Sure, Makes sense.